pkgs/misc/cups/drivers/mfc5890cncupswrapper/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, findutils
+, stdenv
+, fetchurl
+, dpkg
+, makeWrapper
+, coreutils
+, gnugrep
+, gnused
+, mfc5890cnlpr
+, pkgsi686Linux
+, psutils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mfc5890cncupswrapper";
+  version = "1.1.2-2";
+
+  src = fetchurl {
+    url = "https://download.brother.com/welcome/dlf006170/${pname}-${version}.i386.deb";
+    sha256 = "111qggyb18mbspwmidbkxsy55jdfagl66qmrv9zsvz9x3p6b1q2h";
```

Comment:
Can we use an SRI hash and `hash` instead of `sha256` here? Building this package should give you an SRI-hash if you are on a reasonably new nix version.